regulations/templates/regulations/node.html

```diff
@@ -19,34 +19,40 @@
 </div>
 {% endif %}
 
-<div class="node">
-{%if node.marked_up %}
-<p {% if node.is_collapsed %}class="collapsed"{% endif %}>
-{% if node.paragraph_marker %}
-  <span class="stripped-marker">{{node.paragraph_marker}}.</span>
-{% endif %}
-<span class="paragraph-text">
-{% if node.node_type == "appendix" %}
-    {{node.marked_up|safe|linebreaksbr}}
-{% else  %}
-    {{node.marked_up|safe}}
-{% endif %}
-</span>
-</p>
-{% comment %}@TODO use node.comments_calledout{% endcomment %}
-{% if node.accepts_comments and not node.title %}
-  <div
-      class="activate-write"
-      data-section="{{ node.full_id }}"
-      data-label="{{ node.human_label }}">
+<div class="node{% if not node.comments_calledout %} not-called-out{% endif %}">
+
+  {%if node.marked_up %}
+    <p {% if node.is_collapsed %}class="collapsed"{% endif %}>
+
+      {% if node.paragraph_marker %}
+        <span class="stripped-marker">{{node.paragraph_marker}}.</span>
+      {% endif %}
+
+      <span class="paragraph-text">
+      {% if node.node_type == "appendix" %}
+          {{node.marked_up|safe|linebreaksbr}}
+      {% else  %}
+          {{node.marked_up|safe}}
+      {% endif %}
+      </span>
+
+    </p>
+
+    {% if node.accepts_comments and not node.title %}
+    <div
+        class="activate-write"
+        data-section="{{ node.full_id }}"
+        data-label="{{ node.human_label }}">
+
+        <div class="paragraph-comment-icon">
+          <i class="fa fa-pencil-square-o" aria-hidden="true"></i>
```

Comment:
Very minor, but since you're editing this, would you mind making it a `span` instead of a `i`?